crates/ironrdp-session/src/active_stage.rs

```diff
@@ -148,6 +148,11 @@ impl ActiveStage {
         Ok(stage_outputs)
     }
 
+    // tell channels that disconnect or deactivate occurred to reset their state
```

Comment:
style: This could be a docstring.
```suggestion
    // Tells channels that disconnect or deactivate occurred to reset their state.
```

Reply:
ok